mkdocs/docs/protocol/onchain-credential-status.md

```diff
@@ -0,0 +1,170 @@
+# OnChain Credential Status
+
+On-chain identity verification requires the ability to check whether a claim has been revoked using a smart contract. To accomplish this, the claim must include information about the smart contract, such as its address, blockchain, and network. This information can be stored in the `CredentialStatus` structure.
+
+The `CredentialStatus` structure should contain the following fields:
+
+```golang
+type CredentialStatus struct {
+	ID                 string               `json:"id"`
+	Type               CredentialStatusType `json:"type"`
+	RevocationNonce    uint64               `json:"revocationNonce"`
+}
+```
+
+The `id` field is a composite field that contains encoded information in the following format:
+
+`did:[did-method]:[blockchain]:[network]:[id]/credentialStatus?revocationNonce=value&contractAddress={chainID}:{contractAddress}`
+
+`type` - credential type
+
+`revocationNonce` - unique revocation nonce
+
+Example:
+
+```json
```

Comment:
Add one more example, please. Because onchain credential status can look like (without any additional params):
```json
{
"id": "did:polygonid:polygon:main:2qCU58EJgrEMWhziKqC3qNXJkZPY8XCxDSBM4mqPkM/credentialStatus",
"type": "Iden3OnchainSparseMerkleTreeProof2023",
"revocationNonce": "1234"
}
```